tasks/smoke-tests/README.md

```diff
@@ -0,0 +1,61 @@
+# Smoke tests
+
+These smoke tests run against the test project fixture at [../../\_\_fixtures\_\_/test-project](../../__fixtures__/test-project).
+They're a critical part of Redwood's CI.
+
+## Running a smoke test
+
+First, you'll need a test project to run smoke tests against:
+
+```
+yarn build:test-project --link <test project path>
+```
+
+Then, set the `REDWOOD_TEST_PROJECT_PATH` env var to the path of your test project:
+
+```
+REDWOOD_TEST_PROJECT_PATH=<test project path>
+```
+
+Redwood has a mini CLI for running the smoke tests:
+
+```
+yarn smoke-tests [options] [smoke-test..]
+```
+
+If you don't specify any smoke tests to run, it'll prompt you.
+
+Smoke tests are written using [Playwright](https://playwright.dev/).
+You can pass options to the underlying Playwright test command (`npx playwright test`) using `--playwrightOptions`.
+Some common ones are `--debug` (to step through a test), `--headed` (to see what's happening), and `--ui` (to run tests in an interactive UI):
+
+```
+yarn smoke-tests --playwrightOptions="--debug"
+yarn smoke-tests --playwrightOptions="--headed"
+yarn smoke-tests --playwrightOptions="--ui"
+```
+
+See `yarn smoke-tests --help` for more.
+
+If you want to skip the CLI all together, you can `cd` into the directory of the smoke test you want to run and run `npx playwright test` directly:
```

Comment:
Maybe mention that you still need `REDWOOD_TEST_PROJECT_PATH` here when running `npx playwright test`

Reply:
See https://github.com/redwoodjs/redwood/pull/9381/commits/2e48d81ca334fe627759b4fe25fdd7e294fd1aa1